File: 1D array/deletion.py
array = []
size = eval(input("Enter size of array: "))

for i in range(size):
    arrItem = input("Enter array item: ")
    array.insert(i, arrItem)

position = eval(input("Enter position to delete: "))
for i in range(position - 1, size - 1):
    array.insert(i, array[i + 1])
    # array[i] = array[i + 1]

print(array)

File: 1D Array/deletion.c
#include <stdio.h>

int main() {
  int size;

  printf("Enter size of array: ");
  scanf("%d", &size);

  int array[size];

  for (int i = 0; i < size; i++) {
    printf("Enter element %d: ", i + 1);
    scanf("%d", &array[i]);
  };
  printf("\n");

  int position;
  printf("What position do you want to delete: ");
  scanf("%d", &position);

  for (int i = position - 1; i < size; i++) {
    array[i] = array[i + 1];
  }
  size = size - 1;
  printf("\n");

  for (int i = 0; i < size; i++) {
    printf("%d\n", array[i]);
  }

  return 0;
}